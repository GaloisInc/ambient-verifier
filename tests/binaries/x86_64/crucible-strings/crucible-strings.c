#include <stdlib.h>
#include "ambient_assert.h"

int f(const char *x) {
  return 0;
}

void g(char *x) {}

int main() {
  ambient_assert(f("hello") == 1);
  ambient_assert(f("world!") == 2);
  ambient_assert(f("turbo-wombat") == 3);

  char *c = malloc(sizeof(char) * 6);
  g(c);
  ambient_assert(f(c) == 1);
  return 0;
}
