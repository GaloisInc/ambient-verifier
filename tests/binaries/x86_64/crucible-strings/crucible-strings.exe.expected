successful: 0
failed: 0
overrideDir: tests/binaries/x86_64/crucible-strings/overrides
